siliconcompiler/report/dashboard/cli/board.py

```diff
@@ -337,6 +345,11 @@ def stop(self):
         if not self.is_running():
             return
 
+        if self._registered_runs:
+            self._registered_runs -= 1
+            if self._registered_runs:
+                return
```

Comment:
does accesses to `_registered_runs` need to be gated by a lock?

Reply:
`_registered_runs` can be removed if we leave the live display running the entire time.